src/CreditCard/Loading/Loading.css

```diff
@@ -0,0 +1,43 @@
+.lds-ring {
+    display: none;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    width: 60px;
+    height: 100px;
+}
+
+.lds-ring div {
+    box-sizing: border-box;
+    display: block;
+    position: absolute;
+    width: 64px;
+    height: 64px;
+    margin: 8px;
+    border: 8px solid #fff;
+    border-radius: 50%;
+    animation: lds-ring 1.2s cubic-bezier(0.5, 0, 0.5, 1) infinite;
+    border-color: #fff transparent transparent transparent;
```

Comment:
You can also use `border-bottom-color`

https://developer.mozilla.org/en-US/docs/Web/CSS/border#formal_definition

Reply:
also `border: 0 0 8px 0 solid #fff;` and you erase line 18

Reply:
No es necesario porque es el borde de todo el loading